style_editor.html

```diff
@@ -1690,6 +1690,7 @@
   IN_FRAME = false;
 }
 
+var graflexState = true;
```

Comment:
This should just be: var graflexState;
Assigning it to true is confusing, because it's not going to be of type boolean later.

Reply:
indeed. In fact, it can just go away, as it's declared already on page load near SavedState.
Fixed.